.rubocop.yml

```diff
@@ -27,4 +27,6 @@ Style/BlockComments:
   Enabled: false
 Style/StringLiterals:
   Enabled: false
+Metrics/BlockLength:
+  Enabled: false
```

Comment:
Please do not disable the block length check. Methods that are too long should be refactored, in most cases.